app/Http/Requests/Auction/CreateAuctionRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Otoszroto\Http\Requests\Auction;
+
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateAuctionRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
Mimo, że sprawdzamy to middlewarem w routach czy użytkownik jest zalogowany, to fajnie jakbyśmy mieli porządek i tutaj też to sprawdzili
```suggestion
        return auth()->check();
```